index.html

```diff
@@ -188,5 +188,23 @@ <h2 class="presentation2">About <br> myself <br></h2>
 </div>
 </section>
 <!-- about myself finish-->
+  <!-- contact form start -->
+<section class="sec3">
+      <div class="contact">
+          <div class="contact_text">
+          <h2 class="contactme">Contact me</h2>
+          <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+          </div>
+          <form>
+              <input type="text" name="name" placeholder="Yeremias" required>
+              <br>
+              <input type="email" name="email" placeholder="natajayanji14@gmail.com" required>
+              <br>
+              <textarea name="message" cols="30" rows="10" placeholder="Write in your message here"></textarea>
+              <br>
+              <button class="button_touch" type="button">Get In Touch</button>
```

Comment:
- [ ] You should change the `type` property of this button to `submit` because it needs to submit form data to the address present in the `form's action` property upon click.